builds/azure-pipelines/template-steps-build-test.yml

```diff
@@ -15,9 +16,16 @@ steps:
   inputs:
     version: '3.1.x'
 
-- script: npm install -g azure-functions-core-tools@3
+- script: npm install -g azure-functions-core-tools
```

Comment:
This is fine for now, but we should be aware of major changes potentially breaking us. But for now I think it's best to take the latest and handle it later if something does end up breaking because of that.